### PR TITLE
Refs [TASK] [36053] Create UI Detail screen

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#Task 36053: Create UI Detail] (https://edu-redmine.sun-asterisk.vn/issues/36053)
## WHAT
- Create UI for detail screen
## Evidence (Screenshot or Video)
182973095_141273594591270_6634528229787964582_n

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/sun/unsplash_01/ui/detail/ImageDetailFragment.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            false
+        ).apply {
+            lifecycleOwner = this@ImageDetailFragment
+        }
+        return imageDetailBinding.root
+    }
+
+    companion object {
+        fun newInstance() = ImageDetailFragment()
+    }
```
Comment (vvanan158) [inline app/src/main/java/com/sun/unsplash_01/ui/detail/ImageDetailFragment.kt:36]:
```suggestion
    companion object {
    
        fun newInstance() = ImageDetailFragment()
    }
```

File: app/src/main/java/com/sun/unsplash_01/ui/detail/ImageDetailFragment.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import com.sun.unsplash_01.R
+import com.sun.unsplash_01.databinding.FragmentDetailBinding
+import com.sun.unsplash_01.extensions.hideStatusBar
+
+class ImageDetailFragment : Fragment() {
+
+    private lateinit var imageDetailBinding: FragmentDetailBinding
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/unsplash_01/ui/detail/ImageDetailFragment.kt]:
```suggestion
    private lateinit var binding: FragmentDetailBinding
```

File: app/src/main/java/com/sun/unsplash_01/ui/detail/ImageDetailFragment.kt [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            container,
+            false
+        ).apply {
+            lifecycleOwner = this@ImageDetailFragment
+        }
+        return imageDetailBinding.root
+    }
+
+    companion object {
+
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/unsplash_01/ui/detail/ImageDetailFragment.kt]: remove blank

File: app/src/main/res/layout/fragment_detail.xml [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+            android:src="@drawable/ic_draw"
+            app:layout_constraintBottom_toBottomOf="@+id/cardViewAuthorAvatar"
+            app:layout_constraintEnd_toStartOf="@id/imageButtonCrop"
+            app:layout_constraintTop_toBottomOf="@id/imageViewImageDetail"
+            app:layout_constraintTop_toTopOf="@+id/cardViewAuthorAvatar" />
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/textViewDivideItem"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/_1sdp"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_detail.xml:107]: sao height text lại 1dp vậy e?

Comment (ghost) [reply app/src/main/res/layout/fragment_detail.xml:107]: Cái này là em tạo cái thanh màu xám phía trên ở giữa thông tin với cái text saturation đầu tiên ạ

Comment (tuanlvt-1117) [reply app/src/main/res/layout/fragment_detail.xml:107]: sử dụng thẻ View

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**